documentation/src/main/resources/pages/ditto/installation-operating.md

```diff
@@ -88,43 +88,64 @@ proxy_set_header              x-ditto-pre-authenticated "nginx:${remote_user}";
 ### OpenID Connect
 
 The authentication provider must be added to the ditto-gateway configuration.
+`auth-subjects`, an optional field, takes a list of placeholders that will be
+evaluated against incoming JWT's.
+For each entry in `auth-subjects` and authorization subject will be generated.
+If the entry contains unresolvable placeholders, it will be ignored in full.
+When `auth-subjects` is not provided, the “sub” claim (`{{ jwt:sub }}`) is used by default.
+
 
 ```
 ditto.gateway.authentication {
     oauth {
       openid-connect-issuers = {
-        myprovider = "localhost:9000"
+        myprovider = {
+          issuer = "localhost:9000"
+          auth-subjects = [
+            "{{ jwt:sub }}",
+            "{{ jwt:sub }}/{{ jwt:scope }}",
+            "{{ jwt:sub }}/{{ jwt:scope }}@{{ jwt:client_id }}",
+            "{{ jwt:sub }}/{{ jwt:scope }}@{{ jwt:non_existing }}",
```

Comment:
I just learned that the "scope" field in JWT has the key "scp" and not "scope" - so we should adjust the example :)

```suggestion
            "{{ jwt:sub }}/{{ jwt:scp }}",
            "{{ jwt:sub }}/{{ jwt:scp }}@{{ jwt:client_id }}",
            "{{ jwt:sub }}/{{ jwt:scp }}@{{ jwt:non_existing }}",
```

Reply:
Ah, didn't know that. thanks!